addons/mail/static/src/js/followers.js

```diff
@@ -239,30 +239,42 @@ var Followers = AbstractField.extend({
             })
             .then(this._reload.bind(this));
     },
+    /**
+     * Remove partners or channels from the followers
```

Comment:
you might want to specify `ids` in the docstring
```suggestion
     * Remove partners or channels from the followers
     *
     * @param {Object} ids
```